server/routes/api/place.py

```diff
@@ -380,22 +384,60 @@ def get_related_place(dcid,
                                 is_per_capita=is_per_capita)
 
 
-@cache.memoize(timeout=3600 * 24)  # Cache for one day.
+@cache.memoize(timeout=3600 * 24)
+def get_place_cohort(cohort):
+    """Get all the place dcids for a cohort set."""
+    return dc.get_property_values([cohort], 'member')[cohort]
+
+
 @bp.route('/similar/<stats_var>/<path:dcid>')
```

Comment:
cache this perhaps?

Reply:
caching would make the result deterministic so the result would not be random..

Reply:
we want it to be stable-random though, i.e. the list of similar places is varied as you look at different places. but randomness was a side-effect, rather than a feature.

Reply:
one thing forget to mention is the two server call functions used in the endpoint are cached so this endpoint won't have too much latency.